src/ResourceManagement/DataLake.Analytics/DataLakeAnalytics.Tests/TestHelpers/CommonTestFixture.cs

```diff
@@ -44,8 +44,13 @@ public class CommonTestFixture : TestBase
         public string Location = "East US 2";
         public DataLakeAnalyticsManagementHelper DataLakeAnalyticsManagementHelper { get; set; }
         private MockContext context;
-        public CommonTestFixture(MockContext contextToUse, bool createWasbAccount = false)
+        public CommonTestFixture(MockContext contextToUse, bool createWasbAccount = false, bool isDogfood = false)
         {
+            if(isDogfood)
+            {
```

Comment:
This is not a best practice - it creates maintenance headaches, especially as the regions supported in dogfood, and even the regions in Azure that support the service you want change over time.  It's a better practice to use the resource manager library to get the needed provider locatiosn for your service